modules/ROOT/pages/live-booting-ipxe.adoc

```diff
@@ -1,41 +0,0 @@
-= Booting live Fedora CoreOS via iPXE
```

Comment:
I think you missed this one: I think we should keep this page empty and add a link to the new one otherwise older links will ive users a 404.

Reply:
Nope, I [added a redirect](https://github.com/coreos/fedora-coreos-docs/pull/355/files#diff-53f46e8a5da5e47c477534e2acab55e540e0de4762cfc29d9b2e4400f2bb0e8eR2).

Reply:
Great, thanks, had missed that.